### ISSUE TITLE
mv error

### ISSUE THREAD

**Issue by ericclpt** [Microsoft/BashOnWindows#1416]
Hello 

A final "/" in a mv source path make command fail while in a true Linux FS this is OK.
My Windows ver Microsoft Windows [version 10.0.14393]

* Expected results : 
On a plain Linux FS mv normal behave is : 
```
$ mkdir -p a/b/c 
$ mkdir -p b/c/d 
$ touch b/c/d/bla 
$ mv -vi b/c/d/ a/b/c/ 
« b/c/d/ » -> « a/b/c/d » 
$ ll a/b/c/d 
total 0 
-rw-r--r-- 1 toto toto 0 23 nov.  23:31 bla 
$ ll b/c/ 
total 0
```
On WSL FS in the bash tree (not playing with NTFS "mounted" part of the disk)
* Actual results : 
```
$ mv b/c/d/ a/b/c/ 
mv: cannot move ‘b/c/d/’ to ‘a/b/c/d’: No such file or directory
```
If you remove final "/" in source path `mv b/c/d a/b/c/` command works as it must, but if in some build script, packagers or developpers have used such final "/" in source path, command will fail ...

**Comment by ddfznt** [comment]
@ericclpt [Fixed on #14901](https://msdn.microsoft.com/en-us/commandline/wsl/release_notes#build-14901)

**Comment by ericclpt** [comment]
Thanks, 
I'm not an insider. I will wait for this release and one that will fix timer_create issue ...
